From d5dcfd7d202ff384232bef0b097367f36728102d Mon Sep 17 00:00:00 2001
From: Antoine Musso <hashar@free.fr>
Date: Wed, 15 May 2013 22:13:17 +0200
Subject: [PATCH] doc: rename manpage from `zuul` to `zuul-server`

On Debian, all binaries should have an associated manpage. While
building Zuul package we got the following lintian error:

  W: zuul: binary-without-manpage usr/bin/zuul-server

This is easily fixed by making sphinx to name the man page zuul-server.

Change-Id: Ie9fa681654a8e7e8785c1dcfa5ec21ba27ae3049
Reviewed-on: https://review.openstack.org/29287
Reviewed-by: James E. Blair <corvus@inaugust.com>
Reviewed-by: Paul Belanger <paul.belanger@polybeacon.com>
Approved: Clark Boylan <clark.boylan@gmail.com>
Reviewed-by: Clark Boylan <clark.boylan@gmail.com>
Tested-by: Jenkins
---
 doc/source/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index 2e05cc2..6943bd4 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -216,7 +216,7 @@ latex_documents = [
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    ('index', 'zuul', u'Zuul Documentation',
+    ('index', 'zuul-server', u'Zuul Documentation',
      [u'OpenStack'], 1)
 ]
 
-- 
1.8.2.3

